test/datahike/test/time_variance_test.cljc

```diff
@@ -101,12 +102,14 @@
                (d/q query-with-< history-db [:name "Alice"] date)))))
     (testing "print DB"
       (is (= "#datahike/HistoricalDB {:origin #datahike/DB {:max-tx 536870915 :max-eid 4}}"
-             (pr-str (d/history @conn)))))))
+             (pr-str (d/history @conn)))))
+    (d/release conn)))
 
 (deftest test-as-of-db
   (let [cfg (assoc-in cfg-template [:store :id] "test-as-of-db")
         conn (setup-db cfg)
         first-date (now)
+        _ (sleep 10)
```

Comment:
Why is this necessary? 
A comment about that would be good here. Same for `test-since-db`.

Reply:
I added comments. It is necessary when the clocks of the threads/cpus on the testing machine are not fully synchronized.